testutil/server.go

```diff
@@ -173,26 +173,26 @@ type TestServer struct {
 
 // NewTestServer is an easy helper method to create a new Consul
 // test server with the most basic configuration.
-func NewTestServer() (*TestServer, error) {
-	return NewTestServerConfig(nil)
+func NewTestServer(name string) (*TestServer, error) {
```

Comment:
Added a `NewTestServerConfigT` variant which I'm not too happy about but I'll leave refactoring this bit for another time. This will do for now.